x-pack/plugins/uptime/public/components/settings/types.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  IndexActionTypeId,
+  JiraActionTypeId,
+  PagerDutyActionTypeId,
+  ServerLogActionTypeId,
+  ServiceNowActionTypeId,
+  SlackActionTypeId,
+  TeamsActionTypeId,
+  WebhookActionTypeId,
+  // eslint-disable-next-line @kbn/eslint/no-restricted-paths
+} from '../../../../actions/server/builtin_action_types';
```

Comment:
Could we maybe ask Alerting to export these in the future to avoid busting the linter?

Reply:
yeah another idea was to make an an uptime endpoint for this.